frontend/src/AppBuilder/WidgetManager/configs/widgetConfig.js

```diff
@@ -53,6 +53,7 @@ import {
   linkConfig,
   iconConfig,
   boundedBoxConfig,
+  kanbanBoardConfig,
```

Comment:
Remove kanbanBoardConfig, this will add a Kanban board component, but dropping it will cause the component to break.